datacenter/dtr/2.2/guides/admin/configure/deploy-a-cache.md

```diff
@@ -1,57 +1,168 @@
+
+Docker Content Cache is based on Docker Registry, and uses the same configuration
+file format.
+[Learn more about the configuration options](/registry/configuration.md).
+
+The DTR cache extends the Docker Registry configuration file format by
+introducing a new middleware called `downstream` that has three configuration
+options: `blobttl`, `upstreams`, and `cas`:
+
+```none
+# Settings that you would include in a
+# Docker Registry configuration file followed by
+
+middleware:
+  registry:
+      - name: downstream
+        options:
+          blobttl: 24h
+          upstreams:
+            - originhost: <Externally-reachable address for the origin registry>
+              upstreamhosts:
+                - <Externally-reachable address for upstream content cache A>
+                - <Externally-reachable address for upstream content cache B>
+          cas:
+            - <Absolute path to upstream content cache A certificate>
+            - <Absolute path to upstream content cache B certificate>
+```
+
+Below you can find the description for each Content Cache parameter.
+
+<table>
+  <tr>
+    <th>Parameter</th>
+    <th>Required</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>
+      <code>blobttl</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+The TTL for blobs in the cache. This field takes a positive integer and an optional suffix indicating the unit of time. If
+this field is configured, "storage.delete.enabled" must be configured to true. Possible units are:
+      <ul>
+        <li><code>ns</code> (nanoseconds)</li>
+        <li><code>us</code> (microseconds)</li>
+        <li><code>ms</code> (milliseconds)</li>
+        <li><code>s</code> (seconds)</li>
+        <li><code>m</code> (minutes)</li>
+        <li><code>h</code> (hours)</li>
+      </ul>
+    If you omit the suffix, the system interprets the value as nanoseconds.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <code>cas</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+      A list of absolute paths to PEM-encoded CA certificates of upstream registries.
+    </td>
+  </tr>
+<tr>
+  <td>
+    <code>originhost</code>
+  </td>
+  <td>
+    yes
+  </td>
+  <td>
+      An externally-reachable address for the origin registry, as a fully qualified URL.
+  </td>
+</tr>
+<tr>
+  <td>
+    <code>upstreamhosts</code>
+  </td>
+  <td>
+    no
+  </td>
+  <td>
+    A list of externally-reachable addresses for upstream registries for cache chaining. If more than one host is specified, pulls from upstream content caches will be done in round-robin order.
+  </td>
+</tr>
+</table>
 
-## Deploy a DTR cache
+## Deploying a simple Docker Content Cache
 
-You can deploy a DTR cache on any host that has Docker installed. The only
-requirements are that:
+You can deploy a Docker Content Cache on any host that has Docker installed.
+The only requirements are that:
 
 * Users need to have access to both DTR and the cache
 * The cache needs access to DTR
```

Comment:
Do all the caches need access to DTR, or they just need access to their upstreams?